ELK_Listener/install_elk.yaml

```diff
@@ -56,17 +64,37 @@
      - nginx
      - httpd-tools
   - name: Change elasticsearch network host
-    replace: dest=/etc/elasticsearch/elasticsearch.yml regexp='# network.host{{ ":" }} 192.168.0.1' replace=' network.host{{ ":" }} 0.0.0.0'
+    replace:
+      dest: /etc/elasticsearch/elasticsearch.yml
+      regexp: '# network.host{{ ":" }} 192.168.0.1'
+      replace: ' network.host{{ ":" }} 0.0.0.0'
   - name: Need selinux to be permissive
-    command: setenforce 0
+    shell: setenforce 0
   - name: Change kibana server host
-    replace: dest=/opt/kibana/config/kibana.yml regexp='# server.host{{ ":" }} "0.0.0.0"' replace=' server.host{{ ":" }} "localhost"'
+    replace:
+      dest: /opt/kibana/config/kibana.yml
+      regexp: '# server.host{{ ":" }} "0.0.0.0"'
+      replace: ' server.host{{ ":" }} "localhost"'
   - name: Remove server chunk from nginx conf
-    shell: 'head -n33 /etc/nginx/nginx.conf > /etc/nginx/nginx.conf.tmp'
-  - name: Add final closing brace to /etc/nginx/nginx.conf.tmp
-    shell: 'echo "}" >> /etc/nginx/nginx.conf.tmp'
-  - name: Replace nginx conf with new one
-    shell: 'mv -f /etc/nginx/nginx.conf.tmp /etc/nginx/nginx.conf'
+    lineinfile:
```

Comment:
I could change to a template, the reason why I didn't was to avoid requiring the user to download another file.  But if that is not a big deal, I can change to template